### DISCUSSION TITLE
Unexpected behaviour and results from sensor 1 when in focal plane mode

### DISCUSSION THREAD

**Discussion by Surreal62** [srozum/film_camera_tester#45] [Q&A] [answered]
I have the tester up and running and producing good results in general.
However, only issue I have is when using sensor 1 in Focal Plane Horizontal mode, when shutter speed is anything faster than 1/125 sec, no result is displayed.
As an example:
Olympus OM-3Ti set to 1/2000 sec (known good shutter that reliably hits 1/1980 sec on another tester).
Tester running. Fire shutter. No result produced.
Set shutter speed to 1/125 sec. Fire shutter.
Result produced: 1/1980 sec 
Fire shutter again. Result produced: 1/124 sec.
Set shutter to say 1/250 sec, fire, no result.
Set back to 1/125 sec (or anything lower), fire shutter. Result: 1/248 sec.

In simple shutter mode, results display for the full range of speeds up to 1/2000 sec as I have nothing faster to test with.

So it does seem the tester has the captures data of the higher speeds, but 'delays' issuing results until the next test cycle only when 1/125 sec or slower?

I have checked all the settings and can see nothing to explain this 
Every other aspect of the tester is working very well indeed and results I am seeing are very close to other testers and meters I have access to. Excellent device!

**Comment by srozum** [comment]
What you described looks like "shutter capping", when closing curtain moves too fast and at the end of the frame (point "C") there is no slot. You can see it by looking at difference in curtains travel times at last registered speed.

Of course, you could double check that photodiodes perfectly aligned behind the opening holes, but to me it looks like shutter issue. Pay attention to curtains travel times.

**Reply by Surreal62** [reply]
Thanks for the reply - appreciated.
It isn't capping, as it happens with every camera on test, including cameras just serviced and tested on another tester. 
As mentioned, with ANY speed above 1/125 sec the issue occurs. Capping at 1/250 sec is uncommon where the slit is absent - 500th/1000th and up is usual and I have worked on that many I can see that with the naked eye. 
The OM-3Ti I used in my example reads perfectly on my other test equipment (1/125 and below curtain speeds coincide very closely indeed between my other tester and yours. Closing curtain travels very slightly slower than the opening curtain) and shows zero signs of capping or bounce when using film.
Alignment of the photodiodes was double checked prior to fitting and they were secured in place with releasable staking prior to soldering to keep them in place, but I'll pop the enclosure apart and check them again.

**Reply by srozum** [reply]
Ok, please attach photo of the sensor openings and tester screen readings at 1/125, and photo of your Light Unit diffuser.

In meantime, try to test camera against incandescent light bulb, desk lamp for example. It is possible that you used wrong diffuser or something else what reduces full brightness of a light source. It should be around 17EV

**Comment by Surreal62** [comment]
Thanks again. 
The tester is 'jerry rigged' at the moment 
I have the 17EV on the nail with a 12.22V input.
The Fresnel (40mm focal length) is the right way around and is producing a very even field on the genuine Perspex diffuser panel I got in the UK.
Holes in the sensor are drilled to 0.8mm, recesses of 2mm, and when looking carefully the photodiodes can just be seen when light catches them, so locations are correct. (I did put on another front panel I made with 1mm holes temporarily, but got the same issue)
One reading I saw that did stand out from others when a 1/250 sec was recorded:

A: 4.63ms
B: 4.65ms
C: 7us <------- that isn't right
To: 5us
Tc: -5202us

With a Canon F1n (not NEW F-1) at 1/125 sec I see:

A: 8.76ms
B: 8.78ms
C: 8.83ms
S: 1/144
To: 7.55ms
Tc: 7.64ms (too fast as owner turned the tension up 😐)

It is irksome, as everything else is working absolutely superbly. It has to be something I am not 'seeing'! 😂😂😂
 I'll try the tester in FP Vertical mode and check that. 
I will try with a Nikon FM that is recently serviced and again, a known shutter.

**Reply by srozum** [reply]
The first result is clearly bounce or capping. You can double check it by flipping sensor (tester will automatically swap A<->C). It also could help to see if photodiode "C" is properly aligned.

7.5ms for Canon F1n seems normal, you can check table here https://github.com/srozum/film_camera_tester/wiki/Curtains-Travel-Times
Also, does tester works with this Canon on faster speeds?

**Reply by Surreal62** [reply]
The odd results above were one result after an exposure of 1/250sec when exposed again at 1/125 sec., so anything could be happening there with how/when the data was captured/released.
The camera I used does not suffer from bounce at all.

Unfortunately the same issue with the F1 and a Nikon FM in FP Vertical mode.

I have tried with the sensor reversed and get the same issue.

F1n (old mechanical version - not the newer hybrid shutter version) should be about 11ms from memory. 7.5ms for a NEW F1 would be spot-on - I serviced a 50th Anniversary edition one a few weeks back.

**Reply by srozum** [reply]
The "odd results" is exactly how tester behaves during capping/bounce. Sorry, I couldn't help you here unless you show your sensor build. All your explanations are not helpful.

Btw, are you testing with or without lens?

**Comment by Surreal62** [comment]
It will work out in the end! 

The 'odd' results were not from a 'normal' test run - that was displayed as described earlier after a failed run at say 1/500 sec and then became the result of the next teat carried out at 1/125 sec or below. 

Just to discount the Nano, I am putting a brand new one into the tester.

Testing without lens, but also tried with a 55mm f1.2 wide open on the OM-3Ti - no results at any speed from that.

Images of sensor:

![IMG_20231109_203821_resized_20231109_084308906](https://github.com/srozum/film_camera_tester/assets/19474754/966cd3d7-42fe-4d61-83d0-be1c358f198d)
![IMG_20231109_203836_resized_20231109_084309108](https://github.com/srozum/film_camera_tester/assets/19474754/2627bccf-d2f8-4361-8670-98bdcb178ed8)

**Reply by srozum** [reply]
Ok, I see you have rotated the PCB and "A" is "C" if you place probe in a frame opening with cable sticking to the left. Not a problem, as I said, tester automatically swaps A and C.

So the opening on bottom left corner, photodiode "A" in your case, doesn't look very clean.

I hope you followed the assembly instructions: countersink openings on a back, so when photodiode is pressed by a foam against front plate it's lens automatically centered by conical shape of a countersink. 

Since it works correctly on slower speeds, the only suspicion left is assembly quality.


>The 'odd' results were not from a 'normal' test run - that was displayed as described earlier after a failed run at say 1/500 sec and then became the result of the next teat carried out at 1/125 sec or below.

Yes, and that is how tester behaves in case of capping/bouncy shutter. What's not clear? If during first run point "C" not triggered correctly results will not be displayed since cycle is not finished. Second run will finish the cycle and result printed using whatever timings were registered.

**Reply by Surreal62** [reply, marked as answer]
Update from a few weeks back 
A faulty photodiode was the problem. Changed it out and perfect operation.
A superb bit of equipment!
